Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -308,23 +314,43 @@ extension UIResponder {
   }
 }
 
+// MARK: - EpoxyHostingContent
+
+/// The object that is used to communicate changes in the root view to the
+/// `EpoxySwiftUIHostingController`.
+final class EpoxyHostingContent<RootView: View>: ObservableObject {
+
+  // MARK: Lifecycle
+
+  init(rootView: RootView) {
+    _rootView = .init(wrappedValue: rootView)
+  }
+
+  // MARK: Internal
+
+  @Published var rootView: RootView
+}
+
 // MARK: - EpoxyHostingEnvironment
 
 /// The object that is used to communicate values to SwiftUI views within an
 /// `EpoxySwiftUIHostingController`, e.g. layout margins.
 final class EpoxyHostingEnvironment: ObservableObject {
   @Published var layoutMargins = EdgeInsets()
+  @Published var epoxyIntrinsicContentSizeInvalidator = EpoxyIntrinsicContentSizeInvalidator(invalidate: {})
```

Comment:
nit: probably don't need the epoxy prefix here
```suggestion
  @Published var intrinsicContentSizeInvalidator = EpoxyIntrinsicContentSizeInvalidator(invalidate: {})
```